recipes/libcusolver/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set name = "libcusolver" %}
+{% set version = "11.4.2.57" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+{% set major_version = version.split(".")[0] %}
+{% set minor_version = version.split(".")[0] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: c9460db500d0c7d14debb2c8e0d88fc26a39197363f1b000ccf760866f636f12  # [linux64]
+  sha256: bf3c9a090652d011aafe2190c6f26fe108194cf8512d3c571e406054740168bb  # [ppc64le]
+  sha256: 0ca46b73520657f96f921711c05af7053bd398328d510aecddfff2521c000e91  # [aarch64]
+  sha256: c09836df1f43a3496f2b61ad8827af662ace298508869564a40c30d909ba550f  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+
+outputs:
+  - name: libcusolver
+    requirements:
+      run:
+        - libcublas =12.*
+        - libnvjitlink =12.*
```

Comment:
```suggestion
    requirements:
      host:
        - cuda-version {{ cuda_version }}
      run:
        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
        - libcublas =12.*
        - libnvjitlink =12.*
```

Reply:
btw I think we don't need the version constraint for `libcublas` and `libnvjitlink` anymore, now that we pin `cuda-version` tightly?

Reply:
Agreed we can drop this now that every component should be pinning to `cuda-version` we should be getting this transitively.